### ISSUE TITLE
examples wadors does not seem to be doing loading and viewing images

### ISSUE THREAD

**Issue by shelbypereira** [cornerstonejs/cornerstoneWADOImageLoader#275]
Do I need to set something for APIKey?  I can't seem to get theexamples/wadors/index.html page to view an image.  Is there a sample value for apikey and wadors which will work ?  
If sample working values are available perhaps they can be suggested or set as default on the index.html

**Comment by dannyrb** [comment]
![image](https://user-images.githubusercontent.com/5797588/63948805-4fb10200-ca47-11e9-9d35-8e487fce0cba.png)

It looks like the preflight request is failing. This could be a CORS error, or something that's broken due to how GitHub provides assets with it's `raw` URLs. Ideally, we should be hosting and testing against a resource we host and have control of the headers for.

**Comment by shelbypereira** [comment]
Thanks for your feedback.  I am now hosting the images and using DICOMCloud to serve the wado-rs requests.  This solves the CORS error  however I now have a new issue.  At some point  a call is being made to 

Failed to load resource: net::ERR_CONNECTION_REFUSED
index.html:134 Array(1)
index.html:1 Access to XMLHttpRequest at 'http://localhost:44301/wadors/studies/1.8.1982.10121984.2.0.7.637025740873975508/metadata/frames/1' 

which gives me the CORS error below.  However, from what I see there is no official WADO-RS call to /studies/{UID}/metadata/frames/1

see here:
https://www.dicomstandard.org/dicomweb/retrieve-wado-rs-and-wado-uri/

The only call which supports a frame seems to be:
{s}/studies/{study}/series/{series}/instances/{instance}/frames/{frames}

Do I need to implement something more on the server side, or am I making some other mistake?

--------------------------
COPY of CORS Error (but I think this is clearly due to the fact that DICOM cloud does not serve this url).


from origin 'null' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
localhost:44301/wadors/studies/1.8.1982.10121984.2.0.7.637025740873975508/metadata/frames/1:1 Failed to load resource: net::ERR_FAILED
index.html:1 Uncaught (in promise) XMLHttpRequest
